protocol-units/bridge/service/src/actions.rs

```diff
@@ -83,6 +79,14 @@ where
 			amount,
 		} => {
 			let future = async move {
+
+				if recipient.0.len() == 32 {
+					if let Err(e) = movement_utils::fund_recipient(&recipient).await {
+					tracing::error!("Funding failed: {:?}", e);
```

Comment:
You don't need to trace all action execution error are traced [here](https://github.com/movementlabsxyz/movement/blob/78252235e36c45f07bdf806f24809a23d0da7392/protocol-units/bridge/service/src/lib.rs#L282) 
This is here where you can define some specific process when you get and error.